zkevm-circuits/src/evm_circuit.rs

```diff
@@ -513,13 +514,13 @@ mod evm_circuit_stats {
             }
         }
 
-        println!(
+        info!(
```

Comment:
I think we agreed to remove the logs from the table prints as they were in the tests.

Reply:
Year, Sorry for that. I've done it now